requirements.txt

```diff
@@ -1,3 +1,4 @@
 importlib-resources==1.5.0
 pyinotify==0.9.6
 pyyaml >= 3.0
+mypy-extensions==0.4.1
```

Comment:
this repo is in a weird state requirements-wise ;_;

that said, this also needs to be referenced in https://github.com/Yelp/service_configuration_lib/blob/master/setup.py#L27 (but with no pin, or at most a lower bound on whatever mypy-extensions added TypedDict)

requirements.txt will only be used for building a debian package, but the setup.py bit i linked is what pip will use for actually installing this as a normal python package